gpii/node_modules/ontologyHandler/src/ontologyHandler.js

```diff
@@ -106,6 +107,111 @@ URLs - this aspect is taken care of in the preferences server component.
             transformSpecs[filename] = JSON.parse(transformSpec);
         });
         that.options.ontologyTransformSpecs = transformSpecs;
+        // Generate Ontology Routes
+        var edges = [];
+        fluid.each(transformSpecs, function(spec, key) {
+            // The transforms are bidirectional so we add both directed edges
+            edges.push(key.split('-'), key.split('-').reverse());
```

Comment:
Clearer than using an array here (and in the resulting function) will be a 2-element structure with members "source", "target"